zkevm-circuits/src/mpt_circuit.rs

```diff
@@ -147,7 +149,8 @@ impl<F: Field> MPTContext<F> {
         cb: &mut MPTConstraintBuilder<F>,
         idx: usize,
     ) -> RLPItemView<F> {
-        self.rlp_item.create_view(meta, cb, idx, true)
+        // are nibbles string?
+        self.rlp_item.create_view(meta, cb, idx, true, false)
```

Comment:
Nibbles are not part of the MPT spec, just some helper data that's but in those cells. So no need to enforce anything.